react/week4/usersearch/src/components/SearchResult.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext} from "react";
+import UserContext from "./ListUserContext";
+
+function SearchResult() {
+
+const { users, loading, error } = useContext(UserContext);
+
+  if (loading) return <p className="loading">Loading...</p>;
+  if (error) return <p>{error}</p>;
+
+  return (
+    <ul>
+      {!users || users.length === 0 ? (
```

Comment:
this solutions is very good , but there is a better one if you used to implement it like that 
{!users && <li className="result">No results</li> }
{users &&  users.map((user) => {
          return (
            <li key={user.id} className="listing">
               {user.login} <br />
              Repos : <a href={user.repos_url}> {user.repos_url}</a>
            </li>
            );
        }) }